File: Module1/mario.c
#include <cs50.h>
#include <stdio.h>

int main (){
    int height;
    do
    {
         height = get_int("height: ");
        }
        while(height < 1 || height > 8);

        for(int i = 0; i < height; i++){

            for(int j = 0; j < height - 1 - i; j++){
                printf(" ");
            }

            for(int k = 0; k < i + 1; k++){
            printf("#");
        }
        printf("\n");
        }
}

File: pset6/mario.py
def get_int(prompt):
    while True:
        try:
            value = int(input(prompt))
            if 1 <= value <= 8:
                return value
            else:
                pass
        except:
            pass


def main():
    height = get_int("Height: ")

    for i in range(height):
        for j in range(height - 1 - i):
            print(" ", end="")
        for k in range(i + 1):
            print("#", end="")

        print()


main()
